### PR TITLE
Update front page

### PR DESCRIPTION
**Pull request by pjotrp**

### CODE REVIEW

**Comment by pjotrp** [comment]
@prasunanand can you take a look and merge?

**Comment by prasunanand** [comment]
This looks great.

File: source/index.html [hunk 1]
```diff
@@ -70,7 +69,7 @@ <h2 id="acknowledgements">Acknowledgements</h2>
         <a href="http://www.ruby.or.jp/en/news/20141106.html">The Ruby Association Grant 2014</a> awarded to <a href="https://github.com/domitry">@domitry</a> for The Mikon Project.
     </li>
     <li>
-        <a href="http://www.ruby.or.jp/en/news/20151116.html">The Ruby Association Grant 2015</a> awarded to <a href="https://github.com/v0dro">@v0dro</a> for further development of NMatrix plugins. 
+        <a href="http://www.ruby.or.jp/en/news/20151116.html">The Ruby Association Grant 2015</a> awarded to <a href="https://github.com/v0dro">@v0dro</a> for further development of NMatrix plugins.
```
Comment (prasunanand) [inline source/index.html:72]: Grant was awarded to me,  Alish and Udit as well. I will update

**Review by prasunanand — APPROVED** [review]

**Comment by pjotrp** [comment]
The site is not yet updated. Do we need to trigger it? Or is it on master?

**Pull request merged**